e2fsprogs/tests/test.bats

```diff
@@ -0,0 +1,6 @@
+source "${BATS_TEST_DIRNAME}/../plan.sh"
+
+@test "uuidgen exit code check" {
+  run uuidgen -t
```

Comment:
Even after `hab pkg install --binlink --force` this uuidgen in a default studio will be from `core/util-linux` so this test isn't actually checking the uuidgen from `core/e2fsprogs`.

We'll need to add the full package ident to the path for testing.

Example:

```bash
$ hab studio enter


   hab-studio: Importing 'rakops' secret origin key
» Importing origin key from standard input
★ Imported secret origin key rakops-20181017052326.
   hab-studio: Importing 'rakops' public origin key
» Importing origin key from standard input
★ Imported public origin key rakops-20181017052326.
   hab-studio: Exported: HAB_AUTH_TOKEN=[redacted]
   hab-studio: Exported: HAB_ORIGIN=rakops
--> Detected and loading /src/.studiorc

--> Launching the Habitat Supervisor in the background...
    Running: hab sup run 
    * Use 'hab svc start' & 'hab svc stop' to start and stop services
    * Use 'sup-log' to tail the Supervisor's output (Ctrl+c to stop)
    * Use 'sup-term' to terminate the Supervisor
    * To pass custom arguments to run the Supervisor, export
      'HAB_STUDIO_SUP' with the arguments before running
      'hab studio enter'.

--> To prevent a Supervisor from running automatically in your
    Studio, export 'HAB_STUDIO_SUP=false' before running
    'hab studio enter'.


[1][default:/src:0]# hab pkg install --binlink --force /src/e2fsprogs/results/rakops-e2fsprogs-1.44.5-20190205024132-x86_64-linux.hart
» Installing /src/e2fsprogs/results/rakops-e2fsprogs-1.44.5-20190205024132-x86_64-linux.hart
☛ Verifying rakops/e2fsprogs/1.44.5/20190205024132
☛ Verifying core/glibc/2.27/20190115002733
☛ Verifying core/linux-headers/4.17.12/20190115002705
✓ Installed core/glibc/2.27/20190115002733
✓ Installed core/linux-headers/4.17.12/20190115002705
✓ Installed rakops/e2fsprogs/1.44.5/20190205024132
★ Install of rakops/e2fsprogs/1.44.5/20190205024132 complete with 3 new packages installed.
» Binlinking chattr from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked chattr from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/chattr
» Binlinking uuidgen from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked uuidgen from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/uuidgen
» Binlinking mk_cmds from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked mk_cmds from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/mk_cmds
» Binlinking compile_et from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked compile_et from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/compile_et
» Binlinking lsattr from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked lsattr from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/lsattr
» Binlinking e4defrag from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked e4defrag from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/e4defrag
» Binlinking e2undo from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked e2undo from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/e2undo
» Binlinking tune2fs from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked tune2fs from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/tune2fs
» Binlinking fsck.ext3 from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked fsck.ext3 from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/fsck.ext3
» Binlinking badblocks from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked badblocks from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/badblocks
» Binlinking debugfs from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked debugfs from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/debugfs
» Binlinking dumpe2fs from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked dumpe2fs from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/dumpe2fs
» Binlinking e4crypt from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked e4crypt from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/e4crypt
» Binlinking mkfs.ext4 from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked mkfs.ext4 from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/mkfs.ext4
» Binlinking fsck.ext4 from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked fsck.ext4 from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/fsck.ext4
» Binlinking fsck.ext2 from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked fsck.ext2 from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/fsck.ext2
» Binlinking logsave from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked logsave from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/logsave
» Binlinking resize2fs from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked resize2fs from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/resize2fs
» Binlinking mkfs.ext3 from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked mkfs.ext3 from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/mkfs.ext3
» Binlinking blkid from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked blkid from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/blkid
» Binlinking e2label from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked e2label from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/e2label
» Binlinking mklost+found from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked mklost+found from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/mklost+found
» Binlinking mke2fs from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked mke2fs from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/mke2fs
» Binlinking e2fsck from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked e2fsck from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/e2fsck
» Binlinking uuidd from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked uuidd from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/uuidd
» Binlinking mkfs.ext2 from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked mkfs.ext2 from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/mkfs.ext2
» Binlinking e2image from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked e2image from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/e2image
» Binlinking fsck from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked fsck from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/fsck
» Binlinking e2mmpstatus from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked e2mmpstatus from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/e2mmpstatus
» Binlinking filefrag from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked filefrag from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/filefrag
» Binlinking findfs from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked findfs from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/findfs
» Binlinking e2freefrag from rakops/e2fsprogs/1.44.5/20190205024132 into /bin
★ Binlinked e2freefrag from rakops/e2fsprogs/1.44.5/20190205024132 to /bin/e2freefrag


[2][default:/src:0]# hab pkg install --binlink core/which
» Installing core/which
☁ Determining latest version of core/which in the 'stable' channel
w☛ Verifying core/which/2.21/20190115161401
✓ Installed core/which/2.21/20190115161401
★ Install of core/which/2.21/20190115161401 complete with 1 new packages installed.
» Binlinking which from core/which/2.21/20190115161401 into /bin
★ Binlinked which from core/which/2.21/20190115161401 to /bin/which


[3][default:/src:0]# which uuidgen
/hab/pkgs/core/util-linux/2.31.1/20180608101132/bin/uuidgen
```